doc/guides/bluetooth/autopts/autopts-win10.rst

```diff
@@ -0,0 +1,393 @@
+============================
+
+Setup Install Ubuntu 20.4 on `WSL1 <https://docs.microsoft.com/en-us/windows/wsl/install-win10>`_.
+Open PowerShell as Administrator and run:
+
+.. code-block::
+
+   dism.exe /online /enable-feature /featurename:Microsoft-Windows-Subsystem-Linux /all /norestart
+
+Restart Windows. After restart, open Microsoft Store and install Ubuntu 20.4 LTS.
+
+.. image:: install_ubuntu_on_wsl.png
+   :height: 350
+   :width: 500
+   :align: center
+
+Run Ubuntu. You will be asked to create a user account and password:
+
+.. image:: ubuntu_first_launch.png
+   :height: 150
+   :width: 800
+   :align: center
+
+When finished, run commands:
+
+.. code-block::
+
+    sudo apt update
+    sudo apt upgrade
+
+Install python3:
+
+.. code-block::
+
+    sudo apt install python3
+
+Install pip:
+
+.. code-block::
+
+    sudo apt install python3-pip
+
+Install west:
+
+.. code-block::
+
+    pip3 install --user -U west
+
+Export local bin to PATH:
+
+.. code-block::
+
+    echo 'export PATH=~/.local/bin:"$PATH"' >> ~/.bashrc
+
+Reload PATH:
+
+.. code-block::
+
+    source ~/.bashrc
+
+Install cmake:
+
+.. code-block::
+
+    sudo apt install cmake
+
+Go to your zephyrproject:
+
+.. code-block::
+
+    cd /mnt/c/Users/Codecoup/zephyrproject
+
+and then run:
+
+.. code-block::
+
+    west zephyr-export
+    pip3 install --user wheel
+    pip3 install --user -r /mnt/c/Users/codecoup/zephyrproject/zephyr/scripts/requirements.txt
+
+Check if all modules have been installed:
+
+.. code-block::
+
+    pip3 list
+
+If modules still will be missing, just install them with:
+
+.. code-block::
+
+    pip3 install <module_name>
+
+Install Ninja:
+
+.. code-block::
+
+    pip3 install ninja
+
+Go to home:
+
+.. code-block::
+
+    cd ~
+
+and download a toolchain installer:
+
+.. code-block::
+
+    wget https://github.com/zephyrproject-rtos/sdk-ng/releases/download/v0.12.3/zephyr-sdk-0.12.3-x86_64-linux-setup.run
+
+Give permissions to the installer:
+
+.. code-block::
+
+    chmod +x zephyr-sdk-0.12.3-x86_64-linux-setup.run
+
+and run the installer:
+
+.. code-block::
+
+    ./zephyr-sdk-0.12.3-x86_64-linux-setup.run -- -d ~/zephyr-sdk-0.12.3
+
+Copy rules:
+
+.. code-block::
+
+    sudo cp ~/zephyr-sdk-0.12.3/sysroots/x86_64-pokysdk-linux/usr/share/openocd/contrib/60-openocd.rules /etc/udev/rules.d
+
+Restart the Ubuntu machine. You may want to shutdown all WSL consoles from
+Windows's Git Bash:
+
+.. code-block::
+
+    wsl --shutdown
+
+After Ubuntu restart, go to:
+
+.. code-block::
+
+    cd /mnt/c/Users/codecoup/zephyrproject
+
+and test if west can build:
+
+.. code-block::
+
+    west build -p auto -b nrf52840dk_nrf52840 zephyr/tests/bluetooth/tester/
+
+From now on, you can build projects by typing in Windows's Git Bash:
+
+.. code-block::
+
+    wsl -d Ubuntu-20.04 -u codecoup -- bash -c -i "cd /mnt/c/Users/Codecoup/zephyrproject/ ; west build -p auto -b nrf52840dk_nrf52840 zephyr/tests/bluetooth/tester/"
+
+Install nrftools
```

Comment:
Same thing here about making it clear that this is only relevant if the DUT is an nRF board